lib/upperkut/strategies/priority_queue.rb

```diff
@@ -178,7 +178,7 @@ def counter_key
       end
 
       def queue_key
-        "upperkut:priority_queue:#{to_underscore(@worker.name)}"
+        "upperkut:priority_queue:#{to_underscore(@worker.name.dup)}"
```

Comment:
O que acha de escrever um teste maroto pra isso?

Reply:
Specs are broken without this change 🤔  

I can think of 2 options:

1) I could move the 'to_underscore' stuff to a different method, and then test if it works.

```ruby
      def key
        "upperkut:queued:#{worker_name_key}"
      end

      def worker_name_key
        to_underscore(@worker.name.dup)
      end
```

2) make to_underscore work with frozen strings, dup the string there, and add the unit test. I think this is nicer. WDYT?

Reply:
If specs are broken without this change, then, it is all good to me :D